test/test_helper.rb

```diff
@@ -0,0 +1,7 @@
+require "simplecov"
+
+SimpleCov.start do
+end
+
```

Comment:
Make sure to add the test later on

Reply:
Yes. require_relative 'test_helper' is now life.